diff --git a/pkg/util/apptainerconf/config.go b/pkg/util/apptainerconf/config.go
index a3a59e65b..f95db7ed8 100644
--- a/pkg/util/apptainerconf/config.go
+++ b/pkg/util/apptainerconf/config.go
@@ -398,7 +398,6 @@ memory fs type = {{ .MemoryFSType }}
 # If not set, Apptainer will search $PATH, /usr/local/sbin, /usr/local/bin,
 # /usr/sbin, /usr/bin, /sbin, /bin.
 # mksquashfs path =
-{{ if ne .MksquashfsPath "" }}mksquashfs path = {{ .MksquashfsPath }}{{ end }}
 
 # MKSQUASHFS PROCS: [UINT]
 # DEFAULT: 0 (All CPUs)
@@ -432,7 +431,6 @@ mksquashfs procs = {{ .MksquashfsProcs }}
 # If not set, Apptainer will search $PATH, /usr/local/sbin, /usr/local/bin,
 # /usr/sbin, /usr/bin, /sbin, /bin.
 # unsquashfs path =
-{{ if ne .UnsquashfsPath "" }}unsquashfs path = {{ .UnsquashfsPath }}{{ end }}
 
 # SHARED LOOP DEVICES: [BOOL]
 # DEFAULT: no
